booking-up-for-beauty/smunilla/booking_up_for_beauty.go

```diff
@@ -0,0 +1,57 @@
+package booking
+
+import (
+	"fmt"
+	"time"
+)
+
+func parseDate(date string) time.Time {
+	var layouts = []string{
+		"1/2/2006 15:04:05",
+		"January 2, 2006 15:04:05",
+		"Monday, January 2, 2006 15:04:05",
+	}
+
+	for _, layout := range layouts {
+		t, err := time.Parse(layout, date)
+		if err == nil {
+			return t
+		}
+	}
```

Comment:
I didn't think about it. Makes the rest of the code cleaner. Not sure I like it though as parsing dates would be slower than passing the layout in the call.

Reply:
My thought was that we wouldn't necessarily know the format we were getting the dates in (even though the tests don't cover that possibility and only have one format per test)